[FIX] mail: use correct badge color for messaging and activity menu

keshrath opened the pull request:
Description of the issue/feature this PR addresses:

Current behavior before PR:

![image](https://user-images.githubusercontent.com/4851083/101706416-4ce5ef00-3a89-11eb-9ade-0347e6d460b4.png)

Desired behavior after PR is merged:

![image](https://user-images.githubusercontent.com/4851083/101758366-885bda00-3ad8-11eb-94db-58a24f8b85d3.png)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/63117).

alexkuhn reviewed (changes requested):
Hello @keshrath, thank you for the pointing out the issue.

Actually, the intent is to have the green color on counter badge background, so the badge of messaging menu (bubble icon) is the one that is wrong.

The SCSS variable for this color is `$o-enterprise-primary-color`, which both the activity and messaging menu items should use. This is where the color of messaging menu counter badge is defined: https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/messaging_menu/messaging_menu.scss#L108

I let you make the appropriate changes on this PR, then we can merge this commit. Thanks!

alexkuhn reviewed (changes requested):
Please squash to a single commit and use the follow commit title:

`[FIX] mail: use correct badge color for messaging and activity menu`

Also the current PR cannot be merged with our merge tools due to source branch requiring special format for name, `14.0-<branch-name>`, so that all checks work with 14.0.

Either you make another PR with new source branch name, or we cherry-pick your commit and merge it ourselves (naturally while preserving you as authorship of the commit). Which is best for you?

keshrath commented:
I have squashed the commits and changed the commit message. If it's easy for you then feel free to just cherry-pick the commit. For the next PR I know that I have to name the branch accordingly. Thanks

alexkuhn commented:
Moved to https://github.com/odoo/odoo/pull/63298
